index.html

```diff
@@ -0,0 +1,141 @@
+<!DOCTYPE html>
+<html lang="ru">
+
+  <head>
+
+    <meta charset="utf-8">
+    <meta name="keywords" content="Cаша Касперович,Касперович Саша,Касперович Александр,Александр Касперович,Alex Kasper,Alexander Kasperovich, AlexKasper">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+    <meta name="description" content="First acquaintance with junior front-end developer.">
+    <meta name="author" content="Александр Касперович">
+
+    <title>Alex Kasper</title>
+
+    <link href="style.css" rel="stylesheet" type="text/css">
+
+  </head>
+
+<body>
+
+<section>
+    <header class="top">
+      <div class="foto">
+           <img class="img" src="img/me.jpg" alt="AlexKasper">
+      </div>
+       <div class="name">
+        <h1> Alexander  <br> Kasperovich </h1>
+        <h2>Junior front-end developer</h2>
+      </div>   
+    </header>   
+
+    <main>
```

Comment:
-The main element must not appear as a descendant of the section element.
Semantic mistake.